Rubberduck.CodeAnalysis/Inspections/Concrete/ProcedureNotUsedInspection.cs

```diff
@@ -20,25 +20,37 @@ namespace Rubberduck.CodeAnalysis.Inspections.Concrete
     /// <remarks>
     /// Not all unused procedures can/should be removed: ignore any inspection results for 
     /// event handler procedures and interface members that Rubberduck isn't recognizing as such.
+    /// Public procedures of Standard Modules are not flagged by this inspection regardless of
+    /// the presence or absence of user code references.
     /// </remarks>
     /// <example hasResult="true">
     /// <module name="MyModule" type="Standard Module">
     /// <![CDATA[
     /// Option Explicit
     /// 
+    /// 'No inspection result
     /// Public Sub DoSomething()
     ///     ' macro is attached to a worksheet Shape.
     /// End Sub
+    /// 
+    /// 'Flagged by inspection
+    /// Private Sub DoSomethingElse()
+    ///     ' macro is attached to a worksheet Shape.
+    /// End Sub
     /// ]]>
     /// </module>
     /// </example>
     /// <example hasResult="false">
     /// <module name="MyModule" type="Standard Module">
```

Comment:
Nitpick, and not from this PR, but I noticed some examples say `Module1` and others say `MyModule` - I think we should pick one and stick to it... personally I'd prefer `Module1`.

Reply:
Modified examples to join the `Module1` camp.